### PR TITLE
Fixed L-shaped section on the p5 canvas that shows up when switching to code mode in blockly

### PR DESCRIPTION
**Pull request by jmkulwik**
Note: this just fixes the L-shaped section of the screen. The toolbox overlay is fixed by https://github.com/code-dot-org/code-dot-org/pull/28294 

The version bump on P5.play takes a collection of improvements that were made for Dance Party HOC in addition to the bug fix for the L-shaped section of the screen. I tested the full set of improvements against a student project in both game lab and sprite lab, and there were no noticeable differences between prod and this branch. The change in P5.play that fixes the L-shaped section is this: https://github.com/code-dot-org/p5.play/pull/49

BEFORE:
image

AFTER:
image

### CODE REVIEW

**Review by epeach — APPROVED** [review]
Looks good for a temporary fix. In general, I think it's best to not reference specific tickets in comments - these can quickly become outdated or we can move tracking systems and they become meaningless. I usually link the ticket in the PR, but that might just be a personal preference and isn't ship-blocking.

**Comment by jmkulwik** [comment]
Updated to do the full fix rather than the 'bandaid' fix

File: apps/yarn.lock [hunk 1]
```diff
@@ -2804,6 +2816,16 @@ brcast@^3.0.0:
   version "3.0.1"
   resolved "https://registry.yarnpkg.com/brcast/-/brcast-3.0.1.tgz#6256a8349b20de9eed44257a9b24d71493cd48dd"
 
+brfs@^1.4.0:
```
Comment (epeach) [inline apps/yarn.lock:2819]: Looks like there are a lot of changes in yarn.lock that don't seem directly related to bumping a p5.play version. Do you know where these are coming from?

Comment (jmkulwik) [reply apps/yarn.lock:2819]: The p5.play version was behind by 4 versions due to a bunch of work done for HOC 2018 that only went into the Dance Party repo at the time. This takes those 4 version updates as well as the version bump for the bugfix.
Most of the dependencies come from this PR: https://github.com/code-dot-org/p5.play/commit/0dd8722785f1700842259cd0174cc369d3b8d457

Comment (epeach) [reply apps/yarn.lock:2819]: Ahhh, I see. Wow - that's a lot of dependencies!

**Review by epeach — APPROVED** [review]
LGTM - re-approved

**Pull request merged**